### DISCUSSION TITLE
Docker environment not working properly

### DISCUSSION THREAD

**Discussion by victorgso001** [chatwoot/chatwoot#8186] [Contributors]
### Describe the bug

When trying to instantiate Chatwoot environment through docker, it happened a series of errors, including db:creation errors, db:migration errors and rails errors that are linked to babel and some libraries version.

### To Reproduce

1. In an new repository clone, update `.env`;
2. Follow [docker deployment documentation](https://www.chatwoot.com/docs/contributing-guide/environment-setup/docker).

### Expected behavior

For this execution, it was expected to run db creations, migrations and environment preparation. There is a problem in rails entrypoint prevent it to run properly, and migrations and webserver are not working properly, alongside with errors in frontend.

### Environment

app.chatwoot.com

### Cloud Provider

None

### Platform

Browser

### Operating system

Linux Ubuntu 22.04 LTS

### Browser and version

Chrome 118.0.5993.88 64-bit

### Docker (if applicable)

Docker version 24.0.2, build cb74dfc

Docker Compose version v2.19.1

**Comment by pranavrajs** [comment]
Moving this to a discussion it is not evident that there is an error. Can you please share the errors you face when you set it up?
